rosbag2_storage/test/rosbag2_storage/test_plugin.cpp

```diff
@@ -79,4 +79,11 @@ rosbag2_storage::BagMetadata TestPlugin::get_metadata()
   return rosbag2_storage::BagMetadata();
 }
 
+uint64_t TestPlugin::get_bagfile_size() const
+{
+  std::cout << "\nreturning bagfile size\n";
```

Comment:
Is there a reason to keep this debug output? If not, please remove it.

Same below.

Reply:
all tests currently have that print message. If we decide to remove it, we should remove all of them.

Reply:
The rest of the functions in TestPlugin output similarly.